### ISSUE TITLE
CSS: Field overflow

### ISSUE THREAD

**Issue by azmeuk** [nhn/tui.editor#1749]
Since 3.0.2 the bottom of the field is overlapping with the rest of the content, no matter which `height` parameter is set:

![Screenshot 2021-08-11 at 09-46-58 Administration - FAQ](https://user-images.githubusercontent.com/60163/128990865-d7b138a8-1ea6-44ab-a04b-e6a84d516b3a.png)

My configuration:

```typescript
    editor = new Editor({
        el: container,
        height: '550px',
        initialValue: value,
        initialEditType: 'wysiwyg',
        language: 'fr',
        extendedAutolinks: true,
        events: {
            change: (ojb) => value = editor.getMarkdown()
        },
        hideModeSwitch: true,
        plugins: [colorPlugin],
        toolbarItems: [
          [],
          ['heading', 'strike', 'bold', 'italic'],
          ['link', 'table', 'image'],
          ['ul', 'ol', 'indent', 'outdent', 'quote'],
          ['hr'],
        ],
    });
```

**Comment by js87zz** [comment]
@azmeuk 
I can't reproduce it in safari, chrome, ff(MacOS), IE11.
![2021-08-12 10-11-26 2021-08-12 10_11_39](https://user-images.githubusercontent.com/37766175/129123125-7c3b9cdb-87b0-4b51-a13b-0f881dbb34fe.gif)

Can you explain your environment or configuration code?

**Comment by js87zz** [comment]
@azmeuk 
Similar issues have been fixed in `v3.0.3`, so can you check it out?

**Comment by azmeuk** [comment]
Yes, `v3.0.3` fixes the issue!
Thanks
